src/components/DataItemList/DataItemsList.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { css } from '@emotion/css';
+import DataItem, { DataItemsProps } from "../DataItem/DataItem";
+
+export type DataItemsListProps = {
+  itemsList: DataItemsProps[];
+  className?: string;
+}
+
+export const DataItemsList = React.forwardRef<HTMLDivElement, DataItemsListProps>(function DataItemList(
+  {itemsList, className, ...props},
+  ref,
+) {
+
+  const getItemClass = (itemIndex: number) => {
```

Comment:
This function can be placed outside the component

Reply:
It requires itemsList @cingopablo

Reply:
You can also pass the length as an argument. 
In this case it wouldn't make much of a difference, but in general is better to keep them out so to not create a new instance of the function with every re-render

Reply:
Yeah